<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPolylineDialog</class>
 <widget class="QDialog" name="EditPolylineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLocation">
      <attribute name="title">
       <string>Положение</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="EditLocationWidget" name="widgetEditLocation" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPen">
      <attribute name="title">
       <string>Линия</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="EditPenWidget" name="widgetEditPen" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBrush">
      <attribute name="title">
       <string>Заливка</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="EditBrushWidget" name="widgetEditBrush" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPoints">
      <attribute name="title">
       <string>Точки</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Гипербаза</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="EditHyperbaseWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Информация</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditLocationWidget</class>
   <extends>QWidget</extends>
   <header>EditLocationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditHyperbaseWidget</class>
   <extends>QWidget</extends>
   <header>EditHyperbaseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditPenWidget</class>
   <extends>QWidget</extends>
   <header>EditPenWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditBrushWidget</class>
   <extends>QWidget</extends>
   <header>EditBrushWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditPolylineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditPolylineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
